include/nbl/asset/utils/CSPIRVIntrospector.h

```diff
@@ -630,6 +548,17 @@ class NBL_API2 CSPIRVIntrospector : public core::Uncopyable
 					std::fill(m_highestBindingNumbers.begin(), m_highestBindingNumbers.end(), -1);
 				}
 
+				inline bool canSpecializationlesslyCreateDescSetFrom() const
+				{
+					for (const auto& descSet : m_descriptorSetBindings)
+					{
+						auto found = std::find_if(descSet.begin(),descSet.end(),[](const SDescriptorInfo& bnd)->bool{ return bnd.isRuntimeSized();});
+						if (found!=descSet.end())
+							return false;
```

Comment:
its not about being runtime sized, its about having the count from spec-constants